src/lavinmq/config.cr

```diff
@@ -66,6 +66,7 @@ module LavinMQ
     property max_deleted_definitions = 8192         # number of deleted queues, unbinds etc that compacts the definitions file
     property consumer_timeout : UInt64? = nil
     property consumer_timeout_loop_interval = 60 # seconds
+    property consumer_max_per_channel = UInt64::MAX
```

Comment:
Not sure if we should do `0` as some parallel to `infinity` or just use this MAX. I mean, hard to believe someone will ever reach such number of consumers??

Reply:
Or have a reasonable default limit? I don't anyone will/should reach even UInt16::MAX, 65K, but even that might be crazy high. Not sure how much memory one consumer uses?

Reply:
Agreed, considering this limit is by channel it makes sense to limit by a lesser value. Altough the price for such decision (2 vs 8 bytes) weighted against limiting (as weird as it might be) clients implementation - seems to payoff?